test/pylib/util.py

```diff
@@ -133,4 +133,16 @@ async def feature_is_enabled():
     await wait_for(feature_is_enabled, deadline)
```

Comment:
We need a fourth test:
upgrade the cluster so it enables the feature, then try to downgrade one of the nodes. The node should refuse to start.